src/base/static/components/organisms/filter-menu.js

```diff
@@ -0,0 +1,210 @@
+import React, { Component, Fragment } from "react";
+import PropTypes from "prop-types";
+import { NavButton, CloseButton } from "../molecules/buttons";
+import { Image } from "../atoms/imagery";
+import { Link, RegularTitle, LargeLabel } from "../atoms/typography";
+import styled from "react-emotion";
+import mq from "../../../../media-queries";
+import { connect } from "react-redux";
+import {
+  updateFilters,
+  filtersPropType,
+  filtersSelector,
+} from "../../state/ducks/filters";
+import {
+  placeFormsConfigPropType,
+  placeFormsConfigSelector,
+} from "../../state/ducks/forms-config";
+
+import Modal from "react-modal";
+Modal.setAppElement("#main");
+
+const FilterNavButton = styled(linkProps => (
+  <NavButton variant="raised" color="primary" onClick={linkProps.onClick}>
+    {linkProps.children}
+  </NavButton>
+))(() => ({
+  [mq[0]]: {
+    display: "none",
+  },
+  [mq[1]]: {
+    display: "block",
+  },
+}));
+
+const FilterMenuWrapper = styled("div")({
+  display: "flex",
+  flexDirection: "column",
+  width: "100%",
+  height: "100%",
+});
+
+const FilterMenuHeading = styled("div")({
+  display: "flex",
+});
+const FilterMenuTitle = styled(RegularTitle)({
+  textAlign: "center",
+  marginLeft: "auto",
+  marginRight: "auto",
+});
+const FilterOptions = styled("div")({
+  marginLeft: "auto",
+  marginRight: "auto",
+});
+const CategoryFilterOption = styled("div")(props => ({
+  display: "flex",
+  flexDirection: "row",
+  marginBottom: "8px",
+  alignItems: "center",
+
+  background: props.isSelected ? props.theme.bg.highlighted : "clear",
+  color: props.isSelected
+    ? props.theme.text.highlighted
+    : props.theme.text.primary,
+
+  "&:hover": {
+    background: props.theme.bg.highlighted,
+    color: props.theme.text.highlighted,
+  },
+}));
+const CategoryImage = styled(Image)({
+  width: "30px",
+  height: "auto",
+});
+const CategoryLabel = styled(LargeLabel)(props => ({
+  marginLeft: "8px",
+  color: "unset",
```

Comment:
I like this idea of using `unset` to indicate the absence of a property. I think in the past I would just use an empty string for this, which I assume is not a great pattern.

Reply:
Oh, I guess that's actually a value for `color`, haha...